app/src/test/scala/org/alephium/explorer/api/model/ApiModelSpec.scala

```diff
@@ -47,7 +50,17 @@ class ApiModelSpec() extends AlephiumSpec {
        |  "gasAmount": ${tx.gasAmount},
        |  "gasPrice": "${tx.gasPrice}"
        |}""".stripMargin
-      check(tx, expected)
+        check(tx, expected)
+      }
+    }
+    "convert to csv" in {
+      forAll(transactionGen) { tx =>
+        //No inputs or outputs so no addresses nor amounts
+        val expected =
+          s"${tx.hash.toHexString},${tx.blockHash.toHexString},${tx.timestamp.millis},${Instant
+            .ofEpochMilli(tx.timestamp.millis)},,,0,0\n"
+        tx.toCsv(addressGen.sample.get) is expected
```

Comment:
Please add an explicit test too. I wanna see if there will be weird symbols (e.g. comma)